base/applications/iexplore/iexplore.inf

```diff
@@ -16,10 +16,13 @@ HKCR,"CLSID\%CLSID_InternetExplorer%\LocalServer32",,,"""%16422%\Internet Explor
 HKCR,"CLSID\%CLSID_InternetExplorer%\ProgID",,,"InternetExplorer.Application.1"
 HKCR,"CLSID\%CLSID_InternetExplorer%\VersionIndependentProgID",,,"InternetExplorer.Application"
 
-HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,,"shdoclc.dll,-190"
-HKCR,"CLSID\%CLSID_Internet%\Shell",,,"OpenHomePage"
-HKCR,"CLSID\%CLSID_Internet%\Shell\OpenHomePage",,,"Open &Home Page"
-HKCR,"CLSID\%CLSID_Internet%\Shell\OpenHomePage\Command",,,"""%16422%\Internet Explorer\iexplore.exe"""
+HKCR,"CLSID\%CLSID_Internet%",,,"Internet"
+HKCR,"CLSID\%CLSID_Internet%","InfoTip",0x00020000,"@%SystemRoot%\system32\shell32.dll,-22546"
+HKCR,"CLSID\%CLSID_Internet%","LocalizedString",0x00020000,"@%SystemRoot%\system32\shell32.dll,-9222"
+HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,0x00020000,"%16422%\Internet Explorer\iexplore.exe,-1"
```

Comment:
Ideally this should be the globe icon `iexplore.exe,13`, but Wine IE has only one icon. Closest one I can find is shell32 `IDI_SHELL_ENTIRE_NETWORK`:
```suggestion
HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,0x00020000,"%SystemRoot%\system32\shell32.dll,-14"
```